guides/common/assembly_converting-a-host-with-convert2rhel.adoc

```diff
@@ -2,22 +2,19 @@ ifdef::context[:parent-context: {context}]
 
 include::modules/con_converting-a-host-to-rhel-with-convert2rhel.adoc[]
 
-include::modules/proc_installing-convert2rhel.adoc[leveloffset=+1]
+include::modules/proc_importing-convert2rhel-ansible-role.adoc[leveloffset=+1]
 
-ifndef::satellite[]
-include::modules/proc_creating-an-activation-key.adoc[leveloffset=+1]
-endif::[]
-ifdef::satellite[]
-== Creating an Activation Key for the Hosts
+include::modules/proc_assigning-convert2rhel-role-to-host.adoc[leveloffset=+1]
 
-See xref:Creating_an_Activation_Key_{context}[].
-endif::[]
+include::modules/proc_creating-variables-for-convert2rhel-role.adoc[leveloffset=+1]
 
-include::modules/proc_preparing-the-host-group.adoc[leveloffset=+1]
+//include::modules/proc_preparing-the-host-group.adoc[leveloffset=+1]
```

Comment:
If we are not going to include it then I would delete the line instead of commenting it